models/issues/issue_project.go

```diff
@@ -16,13 +16,14 @@ import (
 func (issue *Issue) LoadProject(ctx context.Context) (err error) {
 	if issue.Project == nil {
 		var p project_model.Project
-		if _, err = db.GetEngine(ctx).Table("project").
+		has, err := db.GetEngine(ctx).Table("project").
 			Join("INNER", "project_issue", "project.id=project_issue.project_id").
-			Where("project_issue.issue_id = ?", issue.ID).
-			Get(&p); err != nil {
+			Where("project_issue.issue_id = ?", issue.ID).Get(&p)
+		if err != nil {
 			return err
+		} else if has {
```

Comment:
```suggestion
		} 
		if has {
```
These are two different checks

Reply:
They look the same to me. 🤔

Reply:
Technically you are 💯  correct.

From the readability point of view`if ... else if ... else` should be act on same variable.
Here `if` block is related to `err` so `else if` also point to `err`
and for `has` it should be a different `if` block.